src/insight/alembic/versions/a2198ae60b44_added_version_name_unique_constaint.py

```diff
@@ -0,0 +1,27 @@
+"""Added Result.run_id
+
+Revision ID: a2198ae60b44
+Revises: d2198fd60b0e
+Create Date: 2023-12-13 13:25:17.878689
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a2198ae60b44"
+down_revision = "d2198fd60b0e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column("version", "name", existing_type=sa.VARCHAR(length=50), unique=True)
```

Comment:
Do we need to specify `existing_nullable ` because it was previously specified? See https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.alter_column.params.existing_nullable

Reply:
True, I'll add that